addons/auth_passkey/static/src/views/auth_passkey_key_name_form_view.js

```diff
@@ -0,0 +1,38 @@
+/** @odoo-module **/
+
+import { useService } from "@web/core/utils/hooks";
+import { registry } from "@web/core/registry";
+import { FormController } from "@web/views/form/form_controller";
+import { formView } from "@web/views/form/form_view";
+import { startRegistration } from "../../lib/simplewebauthn.js"
+
+export class PassKeyNameFormController extends FormController {
+    setup() {
+        super.setup();
+        this.orm = useService("orm");
+    }
```

Comment:
Here as well, the whole `setup` block can be deleted, as `this.orm` is not used.